samplesheets/assayapps/generic/plugins.py

```diff
@@ -0,0 +1,175 @@
+"""Assay app plugin for samplesheets"""
+
+from django.conf import settings
+
+from samplesheets.plugins import SampleSheetAssayPluginPoint
+from samplesheets.rendering import SIMPLE_LINK_TEMPLATE
+from samplesheets.utils import get_top_header
+from samplesheets.views import MISC_FILES_COLL, RESULTS_COLL
+
+
+# Local constants
+APP_NAME = 'samplesheets.assayapps.generic'
+RESULTS_REPORTS_COMMENT = 'SODAR Assay Plugin ResRep'
+MISC_FILES_COMMENT = 'SODAR Assay Plugin MiscFiles'
+DATA_COMMENTS = [
+    'SODAR Assay Plugin Data1',
+    'SODAR Assay Plugin Data2',
+    'SODAR Assay Plugin Data3',
+]
+
+
+class SampleSheetAssayPlugin(SampleSheetAssayPluginPoint):
+    """Plugin for generic assays in sample sheets"""
```

Comment:
Minor note, but this should probably say "generic data linking", as "generic assay" sounds like a very specific concept :)

Reply:
good idea